### DISCUSSION TITLE
Empty Layer Detected

### DISCUSSION THREAD

**Discussion by tocguy** [supermerill/SuperSlicer#4066] [Q&A]
SS: 2.5.59.6
Windows: 10 22H2

Been getting some strange behavior when slicing. Occasionally, an "Empty Layer Detected" error appears. I use Solidworks to develop the models and then re-import them and use SWs Mesh Diagnostics to check for errors. Today, this file generated the error in SS.
[Plant Support Web.zip](https://github.com/supermerill/SuperSlicer/files/13897730/Plant.Support.Web.zip)

I ran Netfabb on it and it said that it corrected it but alas no change to the slicing. Same error.

Tried it in Cura 5.4 without error. Hmm...

Switched back to SS and toggled Support OFF completely. Sliced again. No error generated. OK, so toggled Support to ON and sliced again. No error.

Go figure!?!?!?

FWIW, the empty layer was detected between 4.1 and 4.8 which when using the slider revealed that only a tiny bit of support was being placed and zero elsewhere.

I'm interested in others results as this appears to be a recurring scenario both with SS and Prusa although admittedly it is frequently the model that is at the root but for the error to just "go away" like it did by toggling Support is kinda weird.

**Comment by supermerill** [comment]
It's best to open an issue if you have an issue like that, instead of a discussion.
Also, it's better to post a project file (.3mf) from file -> save project as, so i can have all your settings to be able to replicate.

I loaded your part, and as it has a pointy tip, it has nothing on the first layer. I cut the bottom and no more issues.

**Comment by tocguy** [comment]
Hello and thx for looking at this. I neglected to point out that the object was laying flat on its _flattest_ side, i.e., the side with the most contiguous planar surface area aka the large side without the protruding nipple. {tee hee}. Based on your suggestion I have opened [Issue 4067](https://github.com/supermerill/SuperSlicer/issues/4067) and include the 3mf project file there.
